src/Tile/SelectableTileGroup.svelte

```diff
@@ -0,0 +1,48 @@
+<script>
+  /**
+   * Specify the selected tile's
+   * @type {any[]}
+   */
+  export let selectedValues = [];
+
+  /** Set to `true` to disable the tile group */
+  export let disabled = false;
```

Comment:
remove

Reply:
just 8/9?

Reply:
the `disabled` prop

Reply:
Revisiting: Why don't you want to support the disabled prop on the fieldset?